src/python/tools/entrypoint_compiler.py

```diff
@@ -1457,7 +1457,7 @@ def parse_arg(argument, inout):
                 assert not is_column
                 arg_obj = NumericArrayArg(argument, inout)
             elif itemType in ["String", "DataView", "PredictorModel",
-                              "TransformModel", "Node"]:
+                              "TransformModel", "Node", "Char"]:
```

Comment:
>Char [](start = 57, length = 4)

Why this is needed?

Reply:
This is needed because the `CharArrayTermSeparators` field of the WordTokenizer has ItemType==Char which the entrypoint compiler did not support.

---
In reply to: [331220260](https://github.com/microsoft/NimbusML/pull/296#discussion_r331220260) [](ancestors = 331220260)